### PR TITLE
[bugfix](profile) insert into select profile could not build successfully

### PR DESCRIPTION
**Pull request by yiguolei**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yiguolei** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by kaijchen** [comment]
Do we need to change `"OlapTableSink"` to `"VOlapTableSink"` here?

```cpp
Status GroupCommitBlockSink::prepare(RuntimeState* state) {
    RETURN_IF_ERROR(DataSink::prepare(state));
    _state = state;

    // profile must add to state's object pool
    _profile = state->obj_pool()->add(new RuntimeProfile("OlapTableSink"));
    init_sink_common_profile();
    _mem_tracker =
            std::make_shared<MemTracker>("OlapTableSink:" + std::to_string(state->load_job_id()));
    SCOPED_TIMER(_profile->total_time_counter());
    SCOPED_CONSUME_MEM_TRACKER(_mem_tracker.get());
```

File: fe/fe-core/src/main/java/org/apache/doris/common/profile/ProfileTreeBuilder.java [hunk 1]
```diff
@@ -242,7 +246,7 @@ private ProfileTreeNode buildSingleInstanceTree(RuntimeProfile instanceProfile,
                     || profile.getName().startsWith(PROFILE_NAME_VDATA_STREAM_SENDER)
                     || profile.getName().startsWith(PROFILE_NAME_VDATA_BUFFER_SENDER)
                     || profile.getName().startsWith(PROFILE_NAME_DATA_BUFFER_SENDER)
-                    || profile.getName().startsWith(PROFILE_NAME_OLAP_TABLE_SINK)) {
+                    || profile.getName().startsWith(PROFILE_NAME_VOLAP_TABLE_SINK)) {
```
Comment (kaijchen) [inline fe/fe-core/src/main/java/org/apache/doris/common/profile/ProfileTreeBuilder.java:249]: `VOlapTableSinkV2` also starts with `VOlapTableSink`, but I guess it's expected to share this logic?

Comment (yiguolei) [reply fe/fe-core/src/main/java/org/apache/doris/common/profile/ProfileTreeBuilder.java:249]: so ..... rubbish logic...
I will add this case

**Review by kaijchen — APPROVED** [review]
LGTM

**Pull request merged**